java/src/jmri/jmrix/loconet/LocoNetThrottle.java

```diff
@@ -380,7 +383,13 @@ public void notifyChangedSlot(LocoNetSlot pSlot) {
                     !((slotStatus & LnConstants.LOCOSTAT_MASK) == LnConstants.LOCO_IN_USE));
             slotStatus = newStat;
         }
-
+        
+        // It is possible that the slot status change we are being notified of
+        // is the slot being set to status COMMON. In which case the slot just
+        // got set to null. No point in continuing. In fact to do so causes a NPE.
+        if (slot == null)
```

Comment:
Even though this condition has only a single statement, please use braces to surround the statement. (It may be better to encapsulate all the functions in a single ```if (slot != null) { … }``` test and remove this short circuit).